housekeeper/archive/core.py

```diff
@@ -35,5 +38,6 @@ def get_archiveassets(run_id):
     for archive_type in ("data", "result"):
         category = "archive-{}".format(archive_type)
         query = api.assets(run_id=run_id, category=category)
-        asset = query.first()
-        yield archive_type, asset
+        assets = query.all()
+        for asset in assets:
+            yield archive_type, asset
```

Comment:
not sure I follow - what does the query return? Isn't it just both tar-archives for the run?